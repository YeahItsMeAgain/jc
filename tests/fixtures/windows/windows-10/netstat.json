[{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"1031","foreign_port":"1032","local_port_num":1031,"foreign_port_num":1032},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"1032","foreign_port":"1031","local_port_num":1032,"foreign_port_num":1031},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"1316","foreign_port":"1317","local_port_num":1316,"foreign_port_num":1317},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"1317","foreign_port":"1316","local_port_num":1317,"foreign_port_num":1316},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"1318","foreign_port":"1319","local_port_num":1318,"foreign_port_num":1319},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"1319","foreign_port":"1318","local_port_num":1319,"foreign_port_num":1318},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"1320","foreign_port":"1321","local_port_num":1320,"foreign_port_num":1321},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"1321","foreign_port":"1320","local_port_num":1321,"foreign_port_num":1320},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"1322","foreign_port":"1323","local_port_num":1322,"foreign_port_num":1323},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"1323","foreign_port":"1322","local_port_num":1323,"foreign_port_num":1322},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"2132","foreign_port":"2133","local_port_num":2132,"foreign_port_num":2133},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"2133","foreign_port":"2132","local_port_num":2133,"foreign_port_num":2132},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"3199","foreign_port":"3200","local_port_num":3199,"foreign_port_num":3200},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"3200","foreign_port":"3199","local_port_num":3200,"foreign_port_num":3199},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"5481","foreign_port":"5482","local_port_num":5481,"foreign_port_num":5482},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"5482","foreign_port":"5481","local_port_num":5482,"foreign_port_num":5481},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"30785","foreign_port":"30786","local_port_num":30785,"foreign_port_num":30786},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"30786","foreign_port":"30785","local_port_num":30786,"foreign_port_num":30785},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"30787","foreign_port":"30788","local_port_num":30787,"foreign_port_num":30788},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"30788","foreign_port":"30787","local_port_num":30788,"foreign_port_num":30787},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"30797","foreign_port":"30798","local_port_num":30797,"foreign_port_num":30798},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"30798","foreign_port":"30797","local_port_num":30798,"foreign_port_num":30797},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"30799","foreign_port":"30800","local_port_num":30799,"foreign_port_num":30800},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"30800","foreign_port":"30799","local_port_num":30800,"foreign_port_num":30799},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"30825","foreign_port":"30826","local_port_num":30825,"foreign_port_num":30826},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"30826","foreign_port":"30825","local_port_num":30826,"foreign_port_num":30825},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"30842","foreign_port":"30843","local_port_num":30842,"foreign_port_num":30843},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"30843","foreign_port":"30842","local_port_num":30843,"foreign_port_num":30842},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"31337","foreign_port":"31338","local_port_num":31337,"foreign_port_num":31338},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"31338","foreign_port":"31337","local_port_num":31338,"foreign_port_num":31337},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"49673","foreign_port":"49674","local_port_num":49673,"foreign_port_num":49674},{"proto":"TCP","local_address":"127.0.0.1","foreign_address":"api","state":"ESTABLISHED","local_port":"49674","foreign_port":"49673","local_port_num":49674,"foreign_port_num":49673}]