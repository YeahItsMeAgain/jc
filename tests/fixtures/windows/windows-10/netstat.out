
Active Connections

  Proto  Local Address          Foreign Address        State
  TCP    127.0.0.1:1031         api:1032               ESTABLISHED
  TCP    127.0.0.1:1032         api:1031               ESTABLISHED
  TCP    127.0.0.1:1316         api:1317               ESTABLISHED
  TCP    127.0.0.1:1317         api:1316               ESTABLISHED
  TCP    127.0.0.1:1318         api:1319               ESTABLISHED
  TCP    127.0.0.1:1319         api:1318               ESTABLISHED
  TCP    127.0.0.1:1320         api:1321               ESTABLISHED
  TCP    127.0.0.1:1321         api:1320               ESTABLISHED
  TCP    127.0.0.1:1322         api:1323               ESTABLISHED
  TCP    127.0.0.1:1323         api:1322               ESTABLISHED
  TCP    127.0.0.1:2132         api:2133               ESTABLISHED
  TCP    127.0.0.1:2133         api:2132               ESTABLISHED
  TCP    127.0.0.1:3199         api:3200               ESTABLISHED
  TCP    127.0.0.1:3200         api:3199               ESTABLISHED
  TCP    127.0.0.1:5481         api:5482               ESTABLISHED
  TCP    127.0.0.1:5482         api:5481               ESTABLISHED
  TCP    127.0.0.1:30785        api:30786              ESTABLISHED
  TCP    127.0.0.1:30786        api:30785              ESTABLISHED
  TCP    127.0.0.1:30787        api:30788              ESTABLISHED
  TCP    127.0.0.1:30788        api:30787              ESTABLISHED
  TCP    127.0.0.1:30797        api:30798              ESTABLISHED
  TCP    127.0.0.1:30798        api:30797              ESTABLISHED
  TCP    127.0.0.1:30799        api:30800              ESTABLISHED
  TCP    127.0.0.1:30800        api:30799              ESTABLISHED
  TCP    127.0.0.1:30825        api:30826              ESTABLISHED
  TCP    127.0.0.1:30826        api:30825              ESTABLISHED
  TCP    127.0.0.1:30842        api:30843              ESTABLISHED
  TCP    127.0.0.1:30843        api:30842              ESTABLISHED
  TCP    127.0.0.1:31337        api:31338              ESTABLISHED
  TCP    127.0.0.1:31338        api:31337              ESTABLISHED
  TCP    127.0.0.1:49673        api:49674              ESTABLISHED
  TCP    127.0.0.1:49674        api:49673              ESTABLISHED